tools/mkdfs/mkdfs.c

```diff
@@ -1,20 +1,30 @@
 #include <stdio.h>
-#include <stdlib.h>
-#include <dirent.h>
+#include <malloc.h>
```

Comment:
This is reverting https://github.com/DragonMinded/libdragon/pull/36 maybe we can also do this conditionally?

Reply:
Windows might work with stdlib, but this change was made before #36 and I seemed to have missed it. I will have to test.